### PR TITLE
Update script_commands.txt

### PR DESCRIPTION
**Pull request by MarvinLamart**
Actually, without "(" and ")" it triggers an error when script is loaded

### CODE REVIEW

**Comment by dastgir** [comment]
Ohh wait, without () it shouldn't trigger error....

File: doc/script_commands.txt [hunk 1]
```diff
@@ -5905,7 +5905,7 @@ controlling each of the spawned mobs with the unit* commands shown below.
 For example:
 
 	// We'll make a poring which will automatically attack invoking player:
-	.@mobGID = monster "Prontera",150,150,"Poring",PORING,1; // PORING is defined in the mob db and its value is 1002
+	.@mobGID = monster ("Prontera",150,150,"Poring",PORING,1); // PORING is defined in the mob db and its value is 1002
```
Comment (dastgir) [inline doc/script_commands.txt]: Could you possible remove space between 'monster' and '('

**Comment by dastgir** [comment]
So got your point, setting it to variable requires ()
But could you possibly remove the space between monster and ( as I mentioned in commit.

**Comment by MishimaHaruna** [comment]
Also please change `"Prontera"` to `"prontera"` while you're editing that line. `"Prontera"` would be an invalid map ID.

**Comment by MarvinLamart** [comment]
No problem, its done :)

**Comment by dastgir** [comment]
Thanks for contribution :)

**Comment by MarvinLamart** [comment]
It's a pleasure to help at my level x)

**Pull request merged**